src/app/static/src/tests/components/resetConfirmation.test.ts

```diff
@@ -171,6 +171,38 @@ describe("Reset confirmation modal", () => {
         expect(mockNewVersion.mock.calls.length).toBe(1);
     });
 
+    it("pressing enter sets waitingForVersion to true and invokes newVersion action for logged in user", async (done) => {
+
+        const mockContinueEdit = jest.fn();
+        const mockNewVersion = jest.fn();
+        const rendered = mount(ResetConfirmation, {
+            attachToDocument: true, // this is deprecated but typescript doesn't recognise attachTo as a type on mount
+            propsData: {
+                open: false,
+                continueEditing: mockContinueEdit,
+                cancelEditing: jest.fn()
+            },
+            store: createStore(mockNewVersion, {currentUser: 'test.user@example.com'})
+        });
+        rendered.setProps({
+            open: true,
+            continueEditing: mockContinueEdit,
+            cancelEditing: jest.fn()
+        })
+        setTimeout(() => {
+            const okBtn = rendered.findAll("button").at(0)
+            expect(okBtn.element).toBe(document.activeElement);
+            // okBtn.trigger("keyup.enter"); // ideally we'd trigger an enter press here but this doesn't do anything in vue tests; however,...
+            okBtn.trigger("click"); // in the browser, pressing enter on an active element seems to produce the same behaviour as clicking it
```

Comment:
ideally we'd trigger an enter press here but this doesn't do anything in vue tests; however, in the browser, pressing enter on an active element seems to produce the same behaviour as clicking it